app/src/views/full-page-examples/applicant-details/index.mjs

```diff
@@ -19,11 +22,17 @@ export default (app) => {
         .exists()
         .not().isEmpty().withMessage('Enter your date of birth year')
     ],
+
+    /**
+     * @param {import('express').Request} request
+     * @param {import('express').Response} response
+     */
     (request, response) => {
       const errors = formatValidationErrors(validationResult(request))
 
       if (!errors) {
-        return response.render('./full-page-examples/applicant-details/confirm')
+        response.render('./full-page-examples/applicant-details/confirm')
+        return
```

Comment:
**question**: Not blocking, but just want to confirm that this is because the types expect the functions handling the requests to return `undefined`, is that it? 😊

Reply:
Nah in this instance `response.render()` already does nothing

I'm just avoiding the ESLint plugin telling me off for missing a `@returns {void}` tag

Would you rather have the tag + return description?

Reply:
Given we're already typing the functions, I'd rather solve it with an extra JSDoc tag and keep the code simpler, yup 😄  Thanks!